src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -1035,27 +1087,65 @@ def create_lock(name,
         resource_name, name, parameters)
 
 
-def _update_lock_parameters(parameters, level, notes, lock_id, lock_type):
```

Comment:
Why do you remove lock-type from this?  I could have sworn you added it to an example above...

Reply:
the lock type is the level, since there's no point in having both, I removed lock_type